src/coap_tinydtls.c

```diff
@@ -1,7 +1,7 @@
 /*
  * coap_tinydtls.c -- Datagram Transport Layer Support for libcoap with tinydtls
  *
- * Copyright (C) 2016 Olaf Bergmann <bergmann@tzi.org>
+ * Copyright (C) 2016-2020 Olaf Bergmann <bergmann@tzi.org>
```

Comment:
The changes here at least warrant for adding the "and others" or your name, or both.

Reply:
Added my name